homeassistant/components/flo/__init__.py

```diff
@@ -71,6 +74,6 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry):
         )
     )
     if unload_ok:
-        hass.data[DOMAIN].pop(entry.entry_id)
+        hass.data[DOMAIN].pop(CLIENT)
```

Comment:
You can't do this, since there can be multiple config entries.

Reply:
good point!